blockchain/message_pool/src/msg_chain.rs

```diff
@@ -345,7 +347,8 @@ pub(crate) async fn create_message_chains<T>(
     base_fee: &BigInt,
     ts: &Tipset,
     chains: &mut Chains,
-    calico_height: ChainEpoch,
+    _calico_height: ChainEpoch,
```

Comment:
Good. If `calico_height` isn't used anymore, you can delete it.

Reply:
Deleted